docs/faq.md

```diff
@@ -270,3 +270,18 @@ Specifically, we apply these rules when you append to `My Dataset`:
 Otherwise, if only one dataset had this option selected, the merge option will no longer be applied.
 
 You are always welcome to edit these options in `My Dataset` to your liking after appending the additional samples.
+
+## Why are some values different after I regenerate My Dataset?
+
+The Portal only offers data processed using a single version of the [`scpca-nf` workflow](github.com/AlexsLemonade/scpca-nf/) for each sample at any given time.
+If any new features or updates are made to the `scpca-nf` workflow, all data currently on the Portal will be re-processed.
```

Comment:
```suggestion
The Portal only offers data processed using a single version of the [`AlexsLemonade/scpca-nf` workflow](github.com/AlexsLemonade/scpca-nf/) for each sample at any given time.
If any new features or updates are made to the `AlexsLemonade/scpca-nf` workflow, all data currently on the Portal will be re-processed.
```
For consistency?

Reply:
Ha, seems I was making these changes exactly while you were suggesting them :)